GitHub/README.md

```diff
@@ -0,0 +1,44 @@
+## `git` and GitHub usage at Tidepool
+
+At Tidepool, we use `git` for version control, with repositories hosted [on GitHub](https://github.com/tidepool-org/ 'Tidepool on GitHub').
+
+This document describes our guidelines for `git` and GitHub usage.
+
+### Branches and pull requests
+
+In all application and library repositories, we use the `master` branch for production code. Production _releases_ are tied to git [tags](#tags).
+
+When making a change in any repository (whether developing a new feature or fixing a bug), our convention is to create a new branch with the name `<GitHub username>/<short description>` - e.g., `jebeck/new-signup-flow` or `jebeck/bugfix-form-validation`. The backend team sometimes/always(?) uses sprint-based branch names - e.g., `darinkrauss/sprint25` or `darinkrauss/sprint25-clinic-roles`.
+
+All significant changes should be made through a pull request that is approved by another Tidepool developer through a process of code review. We often use LGTM "looks good to me" as our approval sign-off at the conclusion of a review and/or one or more rounds of feedback and changes.
+
+What constitutes a *significant* change? In essence, it's anything that could potentially affect the user experience (though perhaps indirectly). It's probably easier, however, to just list the few things that are *not* significant changes:
+
+- updates to dev-only dependencies
+- changes to dev-only tooling (dev tools like the dev server for client-side apps, test runners, CI configuration)
+- changes to the README or documentation
+
+Only changes such as these that are *not* significant may be pushed to master, and even then you may see some of us opening pull requests for such changes if we either wish to expose the changes to our peers for comment or just to prevent potential conflicts on master if a lot of things are currently in flight. Basically: when in doubt, open a pull request!
+
+Note that the following things, which may *seem* insignificant, are subject to code review (and QA):
+
+- updates to production dependencies
+- changes to production build tooling
+- changes to the production static server (for client apps like blip)
+
+### Rebasing
+
+We follow the general guideline of "don't rewrite history." Rebasing to reorder, squash, or otherwise fix commits locally before a branch or a set of commits has been pushed upstream is perfectly acceptable, but once commits are upstream you shouldn't mess with them (by rebasing or, really, by any other means).
+
```

Comment:
Ah, yes, I believe you have suggested this to me before. I'll put that strategy in as a better suggestion. I would have just left it at "don't rewrite history" but I rebased at least once on the uploader redux migration, so it would have been super hypocritical of me to say *don't do this*.